docs/blog/posts/2024-august-meetup-rca.md

```diff
@@ -0,0 +1,173 @@
+process was done, with discussions happening between the PyTexas Foundation Board
+of Directors, and the PyTexas Organizers. The individuals responsible have not
+been identified, and based on our investigation, it probably will not be possible.
+
+## Root Cause Analysis & Corrective Actions
+
+After a thorough investigation, here are the factors we have identified as issues
+and how we intend to resolve them
+
+### Discord Verification Level was set too low
+At the beginning of this year the Discord Verification of users was set from **High** to **Low**. 
+
+* **Low** Verification Level - Users must have a verified email on their
+Discord Account
+* **High** Verification Level - Users must have a verified email on their
+Discord Account, must be registered on Discord for more than 5 minutes,
+and must have been a member of the PyTexas Discord Server for longer than
+10 minutes
+
+This decision was made to allow users who were new to our Meetup and had never 
+used Discord to be able to create an account and join the Meetup immediately. 
+We had experienced users trying to join a few minutes before the Meetup and being 
+unable to.
+
+**Corrective Action** - The Discord Verification has been set back to **High**.
+We will communicate the need to join 10 minutes prior to the Meetup in future
+communications.
+
+### External Emojis were enabled
+At the creation of the server, a setting was enabled to allow users to use external 
+emojis and stickers in the server. This decision was made to promote cross Discord 
+server communities and is a typical setting to have enabled.
+
+**Corrective Action** - These settings have been disabled. The potential for 
+abuse massively outweighs the good.
+
+### Video permissions are enabled for all users
+At the creation of the server, a setting was enabled to allow users to be able 
+to join chats and share their videos and screens. This decision was made so that 
+people could collaborate in voice chats, bringing that in-person touch to the 
+meetup if they want. Also, sharing screens allows for community members to join 
+and have debugging sessions, which we have observed in the community.
+
+**Corrective Action** - Currently none. We are going to continue to allow users 
+to share their video and screen. Doing so would inhibit how users are currently 
+using the Discord server for good. However, we are exploring verification methods 
+for users so the option isn't available for anyone who joins, but members would 
+have to earn the permission.
+
+### Slido not providing moderation tooling in its free tier
+Slido moderation tools only exist at a high tier of their product. This is honestly,
+annoying. Basic moderation tools shouldn't be behind a pay wall. We used Slido 
+at the 2024 PyTexas Conference, but had purchased a multi-day premium plan. 
+This is not cheap, and currently is not an option for us financially.
+
+**Corrective Action** - We are currently moving off of Slido to Google Forms.
```

Comment:
Maybe worth mentioning something along the lines ofhow this is a temporary fix?

Reply:
(e g. Link to the below?)

Reply:
I think it's fine as it. If they're interested they'll read below.